packages/react-resizable-panels/package.json

```diff
@@ -32,6 +52,11 @@
     "#is-development": {
       "development": "./src/env-conditions/development.ts",
       "default": "./src/env-conditions/production.ts"
+    },
+    "#is-browser": {
+      "browser": "./src/env-conditions/browser.ts",
+      "node": "./src/env-conditions/node.ts",
+      "default": "./src/env-conditions/unknown.ts"
```

Comment:
I like most of this stuff! and I appreciate you trying things out and sharing ideas too!

My main concern here is that this is getting pretty large and easy to mess up. One conditional is 🤷🏼  but two is a lot worse 😁 

https://github.com/bvaughn/react-resizable-panels/blob/f303d9534fdab35edf153b915d334c9eba471e33/packages/react-resizable-panels/package.json#L14-L50

Reply:
I wish there was a more compact way to define/manage this.